test/extended/util/annotate/rules.go

```diff
@@ -24,6 +24,7 @@ var (
 			`\[NodeAlphaFeature:VolumeSubpathEnvExpansion\]`, // flag gate is off
 			`\[Feature:IPv6DualStack.*\]`,
 			`version v1 should create Endpoints and EndpointSlices for Pods matching a Service`, // off by default.
+			`\[Feature:ImmutableEphemeralVolume\]`,                                              // flag gate is off
```

Comment:
do we have a BZ?

Reply:
Why do we need a bz? As per the comment, this feature is not enabled by default.

Reply:
I fear that it will go to beta, we enable the feature, but forget this exclusion. Would create a BZ for node team to reevaluate in 4.6.

Reply:
Ah, gotcha.

Is upstream coordinating test selection with enabled features? If so, why isn't origin doing the same? Having to manually enable/features statically seems like an error-prone exercise, as evidenced by how much obviously outdated cruft remains in this file.

Reply:
Done: https://bugzilla.redhat.com/show_bug.cgi?id=1820422